app/products/calls/screens/call_screen/call_screen.tsx

```diff
@@ -71,9 +82,10 @@ export type Props = {
     fromThreadScreen?: boolean;
 }
 
-const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => ({
+const getStyleSheet = ((theme: CallsTheme) => StyleSheet.create({
```

Comment:
in another component you kept `makeStyleSheetFromTheme` with a `CallsTheme` argument, any reason why not to do the same here?

Reply:
I... Have no idea. :) I think I was experimenting with different ways of doing it, and forgot to clean up. Thanks for catching it.